.github/workflows/ci-pull-request.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
     needs: lint
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/pegasus-kv/thirdparties-bin:ubuntu1804
+      image: apachepegasus/thirdparties-bin:ubuntu1804
```

Comment:
has it updated to `apapche/pegasus`?

Reply:
I'll do it in https://github.com/apache/incubator-pegasus/pull/949, this patch doesn't have to do this work.